python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -96,6 +100,10 @@ def __init__(
             column, index, dtype=dtype, name=name, nan_as_null=nan_as_null
         )
 
+    @property
+    def dtype(self):
```

Comment:
I don't think calling this `dtype` is appropriate. series.dtype returns a numpy type, geopandas returns `GeometryDtype`. They are all scalars.

Reply:
Ah that's a good point.

Reply:
Done